src/collection-view.js

```diff
@@ -165,27 +165,38 @@ Marionette.CollectionView = Marionette.View.extend({
   reorder: function() {
     var children = this.children;
     var models = this._filteredSortedModels();
-    var modelsChanged = _.find(models, function(model) {
+    var modelsAdded = _.find(models, function(model) {
```

Comment:
Might mess up the git history. But adding `_.some` calling the `var` `anyModelsAdded` as I fell it reads better than `someModelsAdded` let me know if you want me to change it to `someModelsAdded`